components/breadcrumb/metadata/breadcrumb.yml

```diff
@@ -57,27 +57,27 @@ examples:
     status: Verified
     markup: |
       <nav>
-        <ul class="spectrum-Breadcrumbs spectrum-Breadcrumbs--sizeL">
+        <ul class="spectrum-Breadcrumbs">
           <li class="spectrum-Breadcrumbs-item">
             <button class="spectrum-ActionButton spectrum-ActionButton--sizeM spectrum-ActionButton--quiet">
```

Comment:
accessibility concern: potentially out of scope since you did not edit these buttons

Having the SVG as the only content of the button and setting aria-hidden"true" results in a button that is empty to screen readers, which I was able to reproduce on Voiceover. Potential solution would be to add a title for the SVG. 

Relevant to all open folder svg buttons. I tried to comment each but might of missed a couple.

Reply:
🎣 Good catch. I started a slack conversation with our Accessibility folks. @jnurthen